const ErrorFallback = () => {
  return <p>에러가 발생했습니다. 잠시 후 다시 시도해주세요.</p>;
};

export default ErrorFallback;
